src/domain/session/rightpanel/RoomDetailsViewModel.js

```diff
@@ -0,0 +1,51 @@
+import {ViewModel} from "../../ViewModel.js";
+import {avatarInitials, getIdentifierColorNumber, getAvatarHttpUrl} from "../../avatar.js";
+
+export class RoomDetailsViewModel extends ViewModel {
+    constructor(options) {
+        super(options);
+        this._room = options.room;
+        this._room.on("change", () => this.emitChange());
```

Comment:
You'll need to unsubscribe in the dispose method you override from ViewModel, which also means you can't use an anonymous lambda here. See RoomViewModel